docs/logging.md

```diff
@@ -0,0 +1,17 @@
+## Application Services Logging
+
+When writing code in application-services, code implemented in Rust, Kotlin, Java, or Swift might have to write debug logs. To do so, one should generally log using the normal logging facilities for the language. Where the logs go depends on the application which is embedding the components.
+
+### Accessing logs when running Fenix
+
+On android, logs go to logcat. Android Studio can be used to view the logcat logs; connect the device over USB and view the Logcat tab at the bottom of Android Studio. Check to make sure you have the right device selected at the top left of the Logcat pane, and the correct process to the right of that. One trick to avoid having to select the correct process (as there are main and content processes) is to choose "No Filters" from the menu on the top right of the Logcat pane. Then, use the search box to search for the log messages you are trying to find.
```

Comment:
> On android, logs go to logcat

Probably note that this is the default / current situation. The motivation behind the rc_log crate was mostly that when the a-c team implemented log sinks other than the one that writes to logcat, we'd use those for free.